roles/activemq/README.md

```diff
@@ -386,6 +386,9 @@ See _Role Variables_ below for additional TLS/SSL settings.
 |`activemq_config_xml`| Broker instance configuration file | `amq-broker.xml` |
 |`activemq_config_override_template`| Filename of custom broker xml configuration file to be deployed | `''` |
 |`activemq_service_override_template`| Filename of custom systemd unit template to be deployed | `''` |
+|`activemq_message_counter_enabled`| Whether to enable the message counters | `false` |
```

Comment:
I think that "Whether" here would imply the use of "or not" (to be idiomatic):

_Whether or not to the message counters is enable._